jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -230,6 +232,78 @@ pipeline {
                 }
             }
         } // end of Premerge Test
+
+        // TODO: support parallel testing for different spark versions
+        stage('Databricks Test') {
+            when {
+                beforeAgent true
+                expression {
+                    !skipped
+                }
+            }
+
+            agent {
+                kubernetes {
+                    label "premerge-test-${BUILD_TAG}"
+                    cloud 'sc-ipp-blossom-prod'
+                    yaml "$pluginPremerge"
+                    workspaceVolume persistentVolumeClaimWorkspaceVolume(claimName: "${PVC}", readOnly: false)
+                    customWorkspace "${CUSTOM_WORKSPACE}"
+                }
+            }
+
+            environment {
+                DATABRICKS_TOKEN = credentials("SPARK_DATABRICKS_TOKEN")
+                DATABRICKS_SECRET = credentials("SPARK_DATABRICKS_SECRET")
+                BUILD_CLUSTER_ID = '1015-114738-tilde12'
```

Comment:
yeah we can't do multiple in parallel if using just 1 cluster.  I have scirpts in progress to allow dynamic creating but I didn't think we wanted databricks build in premerge at all?  @sameerz

Reply:
I am a little worried about adding Databricks to the premerge pipeline because of the amount of time it normally takes to spin up a cluster.  Can we pause on this and keep the Databricks build running as a nightly job?  I think we ought to take the same approach to any of the CSP environments we will test on.